ui/src/app/pages/login/account-creation/account-creation-survey.spec.tsx

```diff
@@ -58,3 +69,28 @@ it('should load existing profile data', async() => {
   // Ethnicity
   expect(wrapper.find('[data-test-id="dropdown-ethnicity"]').prop('value')).toEqual(Ethnicity.HISPANIC);
 });
+
+it('should handle error when creating an account', async() => {
```

Comment:
Will this handle all errors? It looks like it just tests the one?

Reply:
Yes, the code handles any type of error. This test demonstrates the end-to-end flow and validates that the error message is templated into the modal dialog. I don't think explicit tests for different error codes would be worth the redundancy.

Reply:
Sorry, yes, I understood the feature, I was just nitpicking the text on the test. This seems fine then.